sdk/src/beta9/cli/config.py

```diff
@@ -175,3 +179,38 @@ def select_context(name: str, config_path: Path):
     save_config(contexts=contexts, path=config_path)
 
     terminal.success(f"Default context updated with '{name}'.")
+
+
+@management.command(
+    name="export-config",
+    help="Export the current workspace configuration",
+)
+@click.option(
+    "--name",
```

Comment:
Can we just do a positional argument instead? This is simple enough to just need that.